cluster-autoscaler/cloudprovider/azure/azure_cache.go

```diff
@@ -120,6 +120,10 @@ func (m *azureCache) regenerate() error {
 		if err != nil {
 			return err
 		}
+		if instances == nil {
+			klog.V(4).Infof("regenerate: no nodes found for node group %s", ng.Id())
```

Comment:
Is default for the chart set to v(3)? If so are we ok with not seeing this for most clusters?

Reply:
This seems appropriate as a debug-level log event (and matches its complement at L118)